men_team_1/app/views/pairings/show.html.erb

```diff
@@ -1 +1,16 @@
-# <h1><%= @page.mentor_i
\ No newline at end of file
+<h1>Request:</h1>
+
+<% if @request.any?%>
+  <% @request.each do |pairing|%>
+    <%mentee = User.find(pairing.mentee_id)%>
+    <span>
+      <%=mentee.first_name%>
+      <%=button_to 'Accept', edit_pairing_path, method: :get, params: {id: pairing.id, status: 'active'}%>
+      <%=button_to 'Decline', edit_pairing_path, method: :get, params: {id: pairing.id, status: 'decline'}%>
+    </span> 
+  <%end%>
+<% else%>
```

Comment:
This does not affect the functionality of the code, but for readability reasons you should be consistent in your spacing inside these tags. Please always puts spaces after `<%` and before `%>`, e.g. `<% else %>`.